### DISCUSSION TITLE
Missing some schema values in UI

### DISCUSSION THREAD

**Discussion by adjsky** [scalar/scalar#1147] [Ideas]
Hi again! When migrating from redoc, I noticed that some schema values are missing in request bodies and parameters.

For example, let's take this openapi spec:

```json
{
  "openapi": "3.1.0",
  "info": {
    "title": "test",
    "version": "1.0.0",
    "description": "test"
  },
  "paths": {
    "test": {
      "get": {
        "parameters": [
          {
            "in": "query",
            "name": "limit",
            "schema": {
              "type": "integer",
              "minimum": 0,
              "maximum": 100000,
              "default": 100
            }
          }
        ]
      }
    }
  }
}
```

This is how scalar renders example spec:
![image](https://github.com/scalar/scalar/assets/49305219/83df731f-8032-4aca-8d0c-86b565c5e2ae)

And this is how redoc renders example spec:
![image](https://github.com/scalar/scalar/assets/49305219/4339d7a0-4695-4418-8bde-c696d814af5a)

I haven't found any place where i could see schema details, which is not a good thing as it misleads users, especially with regard to default values.

I did a little digging [in the code](https://github.com/scalar/scalar/blob/main/packages/api-reference/src/components/Content/Schema/SchemaProperty.vue), and found out that these values are just not processed/rendered.

I could PR changes if that's welcome.

**Comment by hanspagel** [comment]
Oh yeah, those are just missing! Good find. Contributions are welcome! Hit me up if you need any help to get started.

While you’re on it, I just rewrote the CONTRIBUTING guide (in root). Let me know if this is helpful or something is missing there.

If you don’t find time to send a PR, don’t worry. We’ll add it in the next days then. ✌️ Thanks for reporting!
